packages/sindarian-i18n-cli/src/cli.ts

```
@@ -0,0 +1,187 @@
+#!/usr/bin/env node
+
+import { Command } from 'commander'
```

ESLint

[error] "commander" should be replaced with an alternative package. In your project, we recommend sade. Read more here: https://e18e.dev/docs/replacements/cli-builders

(e18e/ban-dependencies)